components/automate-cli/cmd/chef-automate/license.go

```
@@ -205,20 +200,30 @@ func runLicenseStatusCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func runLicenseStatusCmd(cmd *cobra.Command, args []string) error {
+	connection, err := client.Connection(client.DefaultClientTimeout)
+	if err != nil {
+		return nil
```

this should return the error